index.html

```diff
@@ -2139,25 +2012,26 @@ <h2>The <dfn>record type</dfn> string</h2>
       </td>
      </tr>
      <tr>
-      <td><dfn>[[\ActivatedReaderList]]</dfn></td>
+      <td><dfn>[[\PendingRead]]</dfn></td>
       <td>empty <a>set</a></td>
       <td>
-        A <a>set</a> of {{NDEFReader}} instances.
+        A &lt;|promise:Promise|, |reader:NDEFReader|&gt; tuple where |promise|
+        holds a {{Promise}} and |reader| holds an {{NDEFReader}}.
```

Comment:
Why not `pending` as in "PendingWrite" description?

Reply:
fixed